printer_mods/kageurufu/mgn12-euclid/README.md

```diff
@@ -0,0 +1,24 @@
+Afterburner MGN12 Carriage with Integrated Euclid Probe
+=======================================================
+
+Euclid Probe by [nionio6915 (Github)](https://github.com/nionio6915/Euclid_Probe), https://www.euclidprobe.com/
+
+![Printed mount](./Images/real-bottomview.jpg)
+
+This is a modified version of the MGN12 carriage for a Voron Trident or modded Voron 2.4 with a cutout for mounting the upper Euclid PCB. 
+
+![CAD view of the Euclid Probe embedded in the carriage](Images/cad-bottomview.png)
+
+Other than requiring a Euclid Probe, the only additional parts needed for the MGN12 carriage are two M3 heatset inserts for mounting the upper PCB. 
+
+For the Trident, the probe itself can be mounted [fixed onto the gantry](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Frame%20Mount)
+
+On a V2.4, you have the choice of [gantry mounting](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Frame%20Mount) or [fixed bed mounting](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Bed%20Rail%20Mount). The fixed mounting has the benefit of removing the chance of errantly picking up the probe after reaching a high enough Z. 

```

Comment:
all that text after `#` doesn't seem to actually scroll to the right STL, so maybe just specify the STL names here and link to the STLs directory.